yt/visualization/plot_window.py

```diff
@@ -833,7 +875,26 @@ def set_axes_unit(self, unit_name):
 
     @invalidate_plot
     def toggle_right_handed(self):
-        self._right_handed = not self._right_handed
+        issue_deprecation_warning(
+            "toggle_right_handed has been deprecated, use '.flip_horizontal()' instead.",
+            since="4.1.0",
+            removal="4.2.0",
+        )
+        self.flip_horizontal()
+
+    @invalidate_plot
+    def flip_horizontal(self):
+        """
+        flips the horizontal axis (the image's abscissa)
```

Comment:
Same as above, maybe something like
```suggestion
        Revert the direction of the horizontal axis (the image's abscissa)
```

Reply:
went with "inverts"

Reply:
did you ? I don't see a change here, maybe you forgot to push one last time ?